src/blas/backends/mkl_common/mkl_extensions.cxx

```diff
@@ -0,0 +1,156 @@
+/*******************************************************************************
+* Copyright 2022 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions
+* and limitations under the License.
+*
+*
+* SPDX-License-Identifier: Apache-2.0
+*******************************************************************************/
+
+// Buffer APIs
+
+void gemm_bias(sycl::queue &queue, transpose transa, transpose transb, offset offsetc, int64_t m,
+               int64_t n, int64_t k, float alpha, sycl::buffer<int8_t, 1> &a, int64_t lda,
+               int8_t ao, sycl::buffer<int8_t, 1> &b, int64_t ldb, int8_t bo, float beta,
+               sycl::buffer<int32_t, 1> &c, int64_t ldc, sycl::buffer<int32_t, 1> &co) {
+    blas_major::gemm_bias(queue, transa, transb, offsetc, m, n, k, alpha, a, lda, ao, b, ldb, bo,
```

Comment:
Does new DPC++ API require Intel oneMKL sycl library for MKLCPU backend? Because as far as I remember only LAPACK adds sycl library for MKLCPU backend, all other domains build with only 3 libraries.

Reply:
Here is how sycl library is enabled in LAPACK MKLCPU backend https://github.com/oneapi-src/oneMKL/blob/3fe1f9bb897256d8c4d8fcb0de258b12c17c5a87/src/lapack/backends/mklcpu/CMakeLists.txt#L23

Reply:
Good catch!

I've made similar changes in `src/blas/backends/mklcpu/CMakeLists.txt`. I'm testing now building with only mklcpu backend - this worked for me before probably because I always built with GPU backend enabled, even if I ran tests on CPU backend.

Reply:
With change to `CMakeLists.txt`, a build with only mklcpu backend builds and the tests run.